src/KubernetesClient/KubernetesClientConfiguration.ConfigFile.cs

```diff
@@ -24,6 +24,27 @@ public partial class KubernetesClientConfiguration
         /// </summary>
         public string CurrentContext { get; private set; }
 
+        /// <summary>
+        ///     Initializes a new instance of the <see cref="KubernetesClientConfiguration" /> from config file
+        /// </summary>
+        public static KubernetesClientConfiguration BuildDefaultConfig() {
+            var kubeconfig = Environment.GetEnvironmentVariable("KUBECONFIG");
+            if (kubeconfig != null) {
+                return BuildConfigFromConfigFile(kubeconfigPath: kubeconfig);
+            }
+            if (File.Exists(KubeConfigDefaultLocation)) {
+                return BuildConfigFromConfigFile(kubeconfigPath: KubeConfigDefaultLocation);
+            }
+            var tokenPath = KubernetesClientConfiguration.ServiceaccountPath + KubernetesClientConfiguration.ServiceAccountTokenKeyFileName;
```

Comment:
Prefer using `Path.Combine` to build directory paths.

Reply:
done.